common/policies/policy.go

```diff
@@ -393,3 +400,20 @@ func SignatureSetToValidIdentities(signedData []*protoutil.SignedData, identityD
 
 	return identities
 }
+
+func getCertFromSerializedIdentity(serializedIdentity []byte) (*x509.Certificate, error) {
+	id := &msp.SerializedIdentity{}
+	err := proto.Unmarshal(serializedIdentity, id)
+	if err != nil {
+		return nil, errors.Wrap(err, "unmarshaling serialized identity")
+	}
+	pemBlock, _ := pem.Decode(id.IdBytes)
+	if pemBlock == nil {
```

Comment:
A SerializedIdentity doesn't mean you **have to have** an x509 certificate inside it. 
I suggest we make a function that returns a string to be logged, instead of the certificate.
If the identity has a PEM, it will attempt to parse a certificate. 
If the identity has no PEM, it will simply return the identity bytes in base64.